ts/helpers/filehandler.ts

```diff
@@ -0,0 +1,15 @@
+//handle all zip file actions here
+import { promises as fs } from 'fs';
+import * as xml2js from "xml2js-es6-promise";
+
+export default class FileHandler {
+    public static async parseSlideAttributes(fileName) {
```

Comment:
it's parsing a file and not really attributes within a slide so I'd suggest a rename 'parsePresentationFromFile'

Reply:
but the file name is usually the path to a slide or its layouts

Reply:
filename would point to a file on the system and this function simply parses the contents of that file into consumable JSON .. to the function it doesn't matter whether it's slide or layout